docs/build/apps/wallet/sep7.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: URI Scheme to facilitate delegated signing
+sidebar_position: 80
+---
+
+import Header from "./component/header.mdx";
+import { WalletCodeExample as CodeExample } from "@site/src/components/WalletCodeExample";
+
+<Header />
+
+The [Sep-7] standard defines a way for a non-wallet application to construct a URI scheme that represents a specific transaction for an account to sign. The scheme used is `web+stellar`, followed by a colon.
+Example: `web+stellar:<operation>?<param1>=<value1>&<param2>=<value2>`
+
+## Tx Operation
+
+The tx operation represents a request to sign a specific transaction envelope, with [some configurable parameters].
+
+<CodeExample>
+```typescript
+const sourceAccountKeyPair = "G..."
+const destinationAccountKeyPair = "G...
```

Comment:
```suggestion
const destinationAccountKeyPair = "G..."
```

Reply:
fixed in 566202ca83fc41934a712e196afa6af3f81cd2cd